pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -118,7 +118,7 @@ let
         ] ++ optionals x11Support [ tcl tk xlibsWrapper libX11 ]
     )
     ++ optional zlibSupport zlib
-    ++ optional stdenv.isDarwin CF;
+    ++ optionals stdenv.isDarwin [ coreutils CF ];
```

Comment:
I actually wanted to know that myself. Will nix scan for runtime dependency if its derivation is not in buildInputs?